### PR TITLE
Problem: Missing Jackpot Weekly Count Support

### LINKED ISSUE #296: Problem: Support Jackpot Weekly Count

### PR DESCRIPTION
**Pull request by cdc-Hitesh**
- Weekly Count Support
- Jackpot Counts added in HTTP Handler

Fixes #296

### CODE REVIEW

File: appinterface/projection/crossfire/crossfire.go [hunk 1]
```diff
@@ -30,6 +30,11 @@ type Crossfire struct {
 	phaseTwoStartTime        utctime.UTCTime
 	phaseThreeStartTime      utctime.UTCTime
 	competitionEndTime       utctime.UTCTime
+	JackpotOneStartTime      utctime.UTCTime
```
Comment (foreseaz) [inline appinterface/projection/crossfire/crossfire.go:33]: Keep it lowercase to be consistent with other fields?

File: appinterface/projection/crossfire/crossfire.go [hunk 2]
```diff
@@ -569,6 +604,15 @@ func (projection *Crossfire) updateTxSentCount(
 				return fmt.Errorf("error Incrementing tx sent count: %v", errIncrementingTotal)
 			}
 
+			//Increment count for Weekly Jackpot (If Applicable)
+			if len(weeklyJackpotDBKey) <= 0 {
```
Comment (calvinaco) [inline appinterface/projection/crossfire/crossfire.go:608]:
```suggestion
			if weeklyJackpotDBKey == "" {
```

**Review by calvinaco — CHANGES_REQUESTED** [review]

**Review by foreseaz — APPROVED** [review]
LGTM

File: appinterface/projection/crossfire/crossfire.go (hunk 1 above)
Comment (cdc-Hitesh) [reply appinterface/projection/crossfire/crossfire.go:33]: You are right.

File: appinterface/projection/crossfire/crossfire.go (hunk 2 above)
Comment (cdc-Hitesh) [reply appinterface/projection/crossfire/crossfire.go:596]: I agree with this. Is it because of being better in performance, Just curious to understand?

**Review by calvinaco — APPROVED** [review]

**Pull request merged**